### PR TITLE
Add hint property to field component

### PR DESCRIPTION
**Pull request by nhunzaker**
Fix for #101 

screen shot 2015-08-28 at 8 39 02 am

### CODE REVIEW

**Comment by mzlock** [comment]
To make this input easier for users, do you think you could just have them paste in the YouTube Url and then parse it yourself?  That might be easier than training people what they need to put in here.

**Comment by mzlock** [comment]
:+1:

**Comment by nhunzaker** [comment]
@mzlock that's a fantastic idea! Tracking on #104

File: addons/common/field.jsx [hunk 1]
```diff
@@ -4,27 +4,42 @@
...
     return {
+      hint    : null,
       element : 'input',
       type    : 'text'
     }
   },
 
+  getInitialState() {
+    return {
+      hintId: `hint-col-field-${uid++}`
```
Comment (ltk) [inline addons/common/field.jsx]: seems like it could be nicer to use a more global uuid assignment mechanism for things like this.

**Comment by ltk** [comment]
one minor comment. good stuff :+1:

File: addons/common/field.jsx (hunk 1 above)
Comment (nhunzaker) [inline addons/common/field.jsx]: We could use https://www.npmjs.com/package/uid

**Comment by brianjlandau** [comment]
:+1:

**Pull request merged**